packages/wallets/src/features/cfd/components/ModalTradeWrapper/ModalTradeWrapper.tsx

```diff
@@ -55,7 +57,7 @@ const ModalTradeWrapper: FC<PropsWithChildren<TModalTradeWrapper>> = ({ children
                 return (
                     <div className='wallets-modal-trade-wrapper__footer'>
                         <WalletText align='center' size='sm' weight='bold'>
-                            Download {title} on your phone to trade with the {title} account
+                            <Trans defaults={`Download ${title} on your phone to trade with the ${title} account`} />
```

Comment:
```suggestion
                            <Trans defaults={`Download {{title}} on your phone to trade with the {{title}} account`} values={{ title }} />
```

shouldn't the trans component like this?
ref: https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0

Reply:
acked 🙏🏼